### PR TITLE
add HLT muon packages to the "hlt" category

### PR DESCRIPTION
**Pull request by fwyzard**

### CODE REVIEW

File: categories.py [hunk 1]
```diff
@@ -625,20 +625,33 @@
...
+   "HLTrigger/special",
+   "RecoEgamma/EgammaHLTAlgos",
    "RecoEgamma/EgammaHLTProducers",
-   "HLTrigger/JSONMonitoring",
+   "RecoMuon/L2MuonIsolationProducer",
+   "RecoMuon/L2MuonProducer",
+   "RecoMuon/L2MuonSeedGenerator",
+   "RecoMuon/L3MuonIsolationProducer",
+   "RecoMuon/L3MuonProducer",
+   "RecoMuon/L3TrackFinder",
```
Comment (smuzaffar) [inline categories.py]: @fwyzard , should these RecoMuon/L2* packages belong to ```hlt```?

Comment (fwyzard) [inline categories.py]: I think so, RecoMuon/L2* and RecoMuon/L3* are only used for HLT as far as I know.

@sarafiorendi , @BenjaminRS , can you confirm ?

Comment (BenjaminRS) [inline categories.py]: Yes, the code contained in RecoMuon/L2* and RecoMuon/L3* are for HLT purposes.

Comment (davidlange6) [inline categories.py]: @slava77 - fyi - but I agree these are only used in HLT

Comment (smuzaffar) [inline categories.py]: if no objections from @slava77 then I am happy to merge this

Comment (slava77) [inline categories.py]: these just add the signatures to existing packages.
So, go ahead.

**Pull request merged**